ffi/cpp/docs/Makefile

```diff
@@ -0,0 +1,24 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+# Minimal makefile for Sphinx documentation
```

Comment:
Note, this makefile was autogenerated by Sphinx

Reply:
Suggestion: It would be good to add that comment ("Note, this makefile was autogenerated by Sphinx") to the actual file.